dnf-behave-tests/features/module/enable-errors.feature

```diff
@@ -23,8 +23,8 @@ Scenario: Fail to enable a different stream of an already enabled module (dnf)
     And stderr is
         """
         The operation would result in switching of module 'nodejs' stream '8' to stream '10'
-        Error: It is not possible to switch enabled streams of a module.
-        It is recommended to remove all installed content from the module, and reset the module using 'dnf module reset <module_name>' command. After you reset the module, you can install the other stream.
+        Error: It is not possible to switch enabled streams of a module unless explicitly enabled via configuration option module_allow_switch.
```

Comment:
This also needs to fix the option name to module_stream_switch.

Reply:
Updated, thank you.